packages/runc/runc.spec

```diff
@@ -1,9 +1,9 @@
 %global goproject github.com/opencontainers
 %global gorepo runc
 %global goimport %{goproject}/%{gorepo}
-%global commit 4144b63817ebcc5b358fc2c8ef95f7cddd709aa7
-%global shortcommit 4144b63
-%global gover 1.0.1
+%global commit 52b36a2dd837e8462de8e01458bf02cf9eea47dd
+%global shortcommit 52b36a2
```

Comment:
nit: The only place `shortcommit` is used is in `Release:` and I believe it was an artifact of pre-1.0 release numbering; the project seems to use normal, stable numbering now.  Should we just get rid of this?